packages/orm/src/drivers/PostgresDriver.ts

```diff
@@ -80,17 +82,20 @@ export class PostgresDriver implements Driver<Knex.Transaction> {
     if (em.txn) {
       return fn(em.txn as Knex.Transaction);
     }
-    return this.knex.transaction(async (txn) => {
+    await driverBeforeBegin(em, this.knex);
```

Comment:
Ngl kinda surprised you don't have a `maybePromise` "put these all in a list that compacts something something" trick up your sleeve that avoids the `await` calls 😅 

(Agreed they won't really matter to perf, but I do enjoy our trend of leaning into micro-optimizations, for when we resurrect / ship the joist-vs-other-orms benchmarks.)